bluez-async/src/events.rs

```diff
@@ -47,17 +49,23 @@ pub enum AdapterEvent {
 
 /// Details of an event related to a Bluetooth device.
 #[derive(Clone, Debug, Eq, PartialEq)]
+#[non_exhaustive]
```

Comment:
I suppose this is an advantage of the cargo-release workflow: if master always has a -alpha version number then it can be bumped as part of any PRs that make breaking changes.